### PR TITLE
Add some documentation to installer code

### PR DESCRIPTION
**Pull request by ideaship**

### CODE REVIEW

File: bisdn/installer/install.sh [hunk 1]
```diff
@@ -100,11 +121,11 @@ delete_bisdn_linux_gpt_partition()
     partprobe $blk_dev
 }
 
-# Creates a new partition for the BISDN Linux OS.
+# Creates a new gpt partition for the BISDN Linux OS.
```
Comment (jklare) [inline bisdn/installer/install.sh:124]: we should also update line 145 to echo that we create a gpt partition

File: bisdn/installer/install.sh [hunk 2]
```diff
@@ -173,11 +206,11 @@ delete_bisdn_linux_msdos_partition()
     partprobe $blk_dev
 }
 
-# Creates a new partition for the BISDN Linux OS.
+# Creates a new msdos partition for the BISDN Linux OS.
```
Comment (jklare) [inline bisdn/installer/install.sh:209]: we should also update line 237 to echo that we create an msdos partition

File: bisdn/installer/install.sh [hunk 3]
```diff
@@ -320,10 +363,10 @@ else
     exit 1
 fi
 
-# store installation log in BISDN Linux file system
+# store installation log in BISDN Linux file system (/onie-support-*.tar.bz2)
 onie-support $bisdn_linux_mnt
 
-# activate the installation
+# point bootloader to kernel image (for u-boot: copy it to boot partition, too)
```
Comment (jklare) [inline bisdn/installer/install.sh]:
```suggestion
# point bootloader to kernel image (for u-boot: copy it to boot partition in addition to that)
```

File: mk_onie2 [hunk 4]
```diff
@@ -2,6 +2,10 @@
 #
 #  SPDX-License-Identifier:     GPL-2.0
 
+# This script takes a kernel image and a rootfs to create an ONIE installable
+# BISDN Linux image (*.bin), a list of packagss (*.manifest), and a repo
```
Comment (jklare) [inline mk_onie2]:
```suggestion
# BISDN Linux image (*.bin), a list of packages (*.manifest), and a repo
```

File: bisdn/installer/install.sh (hunk 1 above)
Comment (ideaship) [reply bisdn/installer/install.sh:124]: Done.

File: bisdn/installer/install.sh (hunk 2 above)
Comment (ideaship) [reply bisdn/installer/install.sh:209]: Done.

File: bisdn/installer/install.sh (hunk 3 above)
Comment (ideaship) [reply bisdn/installer/install.sh]: Rephrased.

File: mk_onie2 (hunk 4 above)
Comment (ideaship) [reply mk_onie2]: Thanks, fixed.

**Pull request merged**